docs/tutorials.rst

```diff
@@ -1,6 +1,17 @@
-==============
-User Tutorials
-==============
+
+==================
+Tutorials
+==================
+
```

Comment:
if it is true that all 3 are adopted from https://bitbucket.org/dpat/neuroimaging_core_docs/ -- point to that source here , or add at the opening of each one of those pointer to the original location.